focus-ios/focus-ios-tests/XCUITest/CopyPasteTest.swift

```diff
@@ -62,10 +62,14 @@ class CopyPasteTest: BaseTestCase {
             sleep(1)
         }
         searchOrEnterAddressTextField.tap()
-        searchOrEnterAddressTextField.press(forDuration: 1.5)
+        searchOrEnterAddressTextField.tap()
```

Comment:
On iOS 15 and 16, if I press for too long, the menu disappears.

Reply:
Does this still work for iOS 17?

Reply:
Yes, a `tap()` works on iPhone 15 iOS 17 (both iPhone and iPad).